src/components/Candidates/CandidatesJoinUs/CandidatesOpenPositions/FormsModal/FormsModal.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import { Form, FormGroup, Label,Col, Input,Row, Modal, ModalHeader, ModalBody, Button} from 'reactstrap';
+import './FormsModal.css';
+import emailjs from 'emailjs-com';
+
+const service_id = 'juarezid_1234_4321_01';
+const template_id = 'template_6owvhzc';
+const user_id = 'user_exlhFGfiCpajm8iHQostc';
```

Comment:
Global constants need to be all in upper camel case. [JavaScript Code Style guide](https://google.github.io/styleguide/jsguide.html#naming-constant-names)